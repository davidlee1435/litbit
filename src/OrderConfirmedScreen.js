import React from 'react';
import { Image, TouchableHighlight, Picker, TextInput, Button, StyleSheet, Text, View, FlatList } from 'react-native';
import * as firebase from 'firebase';

const _ = require('lodash');

export default class OrderConfirmedScreen extends React.Component {
  static navigationOptions = {
    title: "Order Confirmed",
    headerLeft: null,
  };

  constructor() {
    super();
  }

  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.titleText}>
          Your order is on its way!!
        </Text>

        <FlatList
        data={[{title: 'Title Text', key: 'item1'}, {title: 'title2', key: 'item2'}]}
        renderItem={({item}) => <Text key={item.key}> {item.title}</Text>}
        />

        <View style={styles.checkoutWrapper}>
          <Button
            style={styles.checkoutButton}
            title="Rando button"
            color="#841584"
            accessibilityLabel="Rando Button"
          />
        </View>
      </View>
	  )
  }
}
const styles = StyleSheet.create({
  container: {
    padding: 15,
    alignItems: 'center',
    justifyContent: 'center',
    flex: 1,
  },
  feed: {
    alignItems: 'center',
  },
  itemButtonContainer: {
    alignItems: 'center',
  },
  itemButton: {
    width: 200,
  },
  checkoutButton: {
  },
  checkoutWrapper: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    padding: 15,
  },
    baseText: {
    fontFamily: 'Cochin',
  },
  titleText: {
    fontSize: 20,
    fontWeight: 'bold',
  }
});
