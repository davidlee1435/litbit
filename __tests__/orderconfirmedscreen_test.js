// Link.react-test.js
import React from 'react';
import renderer from 'react-test-renderer';
import OrderConfirmedScreen from '../src/OrderConfirmedScreen.js';

test('OrderConfirmedScreen test', () => {
  const tree = renderer.create(<OrderConfirmedScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});