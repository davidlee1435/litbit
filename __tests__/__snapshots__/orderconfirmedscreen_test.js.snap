// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderConfirmedScreen test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "padding": 15,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Hello!!
  </Text>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "padding": 15,
      }
    }
  >
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Rando Button"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#841584",
              },
            ]
          }
        >
          Rando button
        </Text>
      </View>
    </View>
  </View>
</View>
`;
